pkg/virt-controller/watch/vm_test.go

```diff
@@ -3993,6 +4000,7 @@ func DefaultVirtualMachineWithNames(started bool, vmName string, vmiName string)
 	vmi.Status.Phase = virtv1.Running
 	vmi.Finalizers = append(vmi.Finalizers, virtv1.VirtualMachineControllerFinalizer)
 	vm := VirtualMachineFromVMI(vmName, vmi, started)
+	vm.Finalizers = append(vm.Finalizers, virtv1.VirtualMachineControllerFinalizer)
```

Comment:
WDYT about adding the finalizer only if `started == true`? I think it simulates real life better, right?

Reply:
The finalizer will be added for both started and stopped vm, so from a real life simulation perspective IMHO is better to add it always :slightly_smiling_face: Let me know, Thanks